ideScripts/updateLaunchConfig.py

```diff
@@ -152,10 +152,8 @@ def getDebugLaunchConfig(self):
 
         jsonConfigurationData["name"] = tmpStr.launchName_Debug
         jsonConfigurationData["executable"] = buildData[self.bStr.targetExecutablePath]
-        jsonConfigurationData["svdFile"] = buildData[self.bStr.stm32svdPath]
-        jsonConfigurationData["configFiles"] = []
-        jsonConfigurationData["configFiles"].append(buildData[self.bStr.openOCDInterfacePath])
-        jsonConfigurationData["configFiles"].append(buildData[self.bStr.openOCDTargetPath])
+        jsonConfigurationData["svdFile"] = buildData[self.bStr.stm32SvdPath] + "/" + buildData[self.bStr.stm32SvdFile]

```

Comment:
Although it is OK in this case, since paths are checked and converted previously, I would still prefer using `os.path.join()` method and `pathWithForwardSlashes()`. Also, I avoid multiline code segments if possible.
Anyway, not a big deal, just a note for further (hopefully) participation.

Reply:
Not sure why I didn't `os.path.join()` here, I did it elsewhere, will fix